README.md

```diff
@@ -87,6 +87,12 @@ have been created and contain a ProvisionAccount role (ideally create via the
 
 None
 
+## Notes ##
+
+Running `pre-commit` requires running `terraform init` in every directory that
+contains Terraform code. In this repository, these are the main directory and
+every directory under `examples/`.
```

Comment:
I believe it is only the main directory for this project.

Reply:
Keen eyes there - thanks!  Fixed in https://github.com/cisagov/cool-users-non-admin/pull/2/commits/520e92d2364587b756238803d03a2b06fda34498.